pages/common/tailscale-up.md

```diff
@@ -0,0 +1,36 @@
+# tailscale up
+
+> Connects the client to the Tailscale network.
+> More information: <https://tailscale.com/kb/admin/>.
+
+- Connect to Tailscale:
+
+`sudo tailscale up`
+
+- Connect and offer the current machine to be an exit node for the internet traffic:
+
+`sudo tailscale up --advertise-exit-node`
+
+- Connect using a specific node for the internet traffic:
+
+`sudo tailscale up --exit-node={{exit_node_ip}}`
+
+- Connect and block incoming connections to the current node:
+
+`sudo tailscale up --shields-up`
+
+- Connect and accept DNS configuration from the admin panel:
+
+`sudo tailscale up --accept-dns={{true|false}}`
+
+- Connect and configure Tailscale as a subnet router:
+
+`sudo tailscale up --advertise-routes={{10.0.0.0/24,10.0.1.0/24}}`
+
+- Connect and accept subnet routes from Tailscale:
+
+`sudo tailscale up --accept-routes`
+
+- Reset unspecified settings to their default values and connect:
```

Comment:
So when I example use `--accept-dns=true` this value will be saved until I call `--reset`? I think we should mention that in the description.

Reply:
I'm not sure how the CLI args are saved, but I tried a connection with `--exit-node` and then I had to use `--reset` because without `--exit-node` the connection wasn't working.

Reply:
Hmmm, that is very odd behaviour. What about this?
```suggestion
- Reset unspecified settings to their default values and connect (CLI arguments are persisted for future calls):
```

Reply:
I found this: https://github.com/tailscale/tailscale/issues/1927#issuecomment-845083085

> in 1.8 the CLI saves the options it has been invoked with, and will only change them if explicitly told to do so. In prior releases if one had used: tailscale up --accept-routes and then later used tailscale up --advertise-exit-node, the --accept-routes would be silently discarded and this led to bad outcomes.
> 
> You should be able to use tailscale up --advertise-exit-node=false to explicitly clear the flag if it is no longer wanted.

Reply:
Sounds good @marchersimon